### ISSUE TITLE
Latest release doesn't build

### ISSUE THREAD

**Issue by kpcyrd** [polachok/pnetlink#41]
I've pulled the latest release from crates.io and the project fails to compile:
```
$ cargo build
    Updating registry `https://github.com/rust-lang/crates.io-index`
error: Package `pnet_macros v0.21.0` does not have these features: `with-syntex`
$
```
It seems the current master compiles fine (with a large amount of warnings though), could you consider releasing a new version?

**Comment by rabbitstack** [comment]
I hit the exactly same issue. Would really appreciate if you could fix this soon.

**Comment by rabbitstack** [comment]
Any news on this?

**Comment by polachok** [comment]
I just published 0.0.3

**Comment by rabbitstack** [comment]
I just fetched `0.0.3` successfully. Thanks.
